### PR TITLE
Add serverless-output-to-env plugin

### PR DESCRIPTION
**Pull request by williamsandonz**

### CODE REVIEW

**Comment by williamsandonz** [comment]
Hi @eahefnawy , new plugin here ready for merge !

File: README.md [hunk 1]
```diff
@@ -353,6 +353,7 @@ Add your plugin to [the `plugins.json` file](https://github.com/serverless/commu
...
 | **[Website Domain - `serverless-website-domain`](https://github.com/williamsandonz/serverless-website-domain)** <br/> by [williamsandonz](http://github.com/williamsandonz) <br/> A plugin that creates Route 53 records that point to your Cloudfront hosted static website, including www/non-www redirects. | ![Github Stars](https://img.shields.io/github/stars/williamsandonz/serverless-website-domain.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-website-domain.svg?label=Downloads&style=for-the-badge)|
+| **[Output to ENV - `serverless-output-to-env`](https://github.com/williamsandonz/serverless-output-to-env)** <br/> by [williamsandonz](http://github.com/williamsandonz) <br/> A plugin that creates Route 53 records that point to your Cloudfront hosted static website, including www/non-www redirects. | ![Github Stars](https://img.shields.io/github/stars/williamsandonz/serverless-output-to-env.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-output-to-env.svg?label=Downloads&style=for-the-badge)|
```
Comment (pgrzesik) [inline README.md:356]: The description here does not look correct - could you please look into it? Thanks in advance :bow:

**Comment by williamsandonz** [comment]
Apologies @pgrzesik that's fixed now!

**Review by pgrzesik — APPROVED** [review]
No worries and thank you :bow:

**Pull request merged**